class GroupNotFoundException(Exception):
    pass


class UserNotFoundException(Exception):
    pass
